src/utils/__tests__/generateKeyInsightsData.test.js

```diff
@@ -0,0 +1,78 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2021 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+import generateKeyInsightsData from "../generateKeyInsightsData";
+
+describe("generateKeyInsightsData.js", () => {
```

Comment:
Are you planning to add more test cases for the other functions in this file that are not tested here yet?

Reply:
Yes. But maybe we should at first decide what to do with `v2-jails`?